### ISSUE TITLE
Hovering over the "x"/ cancel transfer button doesn't indicate this button is to cancel a transfer. EOS 6 beta 1

### ISSUE THREAD

**Issue by KarkanAlzwayed** [elementary/files#1740]
## Problem
I don't know if the title is worded correctly or not (English is my second language and I do lack vocabulary at times ), nor do I know if this is a big deal to you guys or not, but I was getting ready to move over to beta 2 and was attempting to transfer my home folder onto an external storage to copy and paste it onto beta 2. Copied the whole home folder and pasted it into the external storage and the normal little window with the progress bar was showing. Hovered over the "x" circular button expecting a tiny box with "cancel" text in it, but there wasn't any. I don't know if that was intentional or not, but I am a person who like small details and wanted to bring that to your attention :)
here is a screenshot: (Note that the cursor is on the circular "x" but for some reason, the screenshot doesn't capture the cursor)

![Screenshot from 2021-06-16 20-59-08](https://user-images.githubusercontent.com/49320606/122314202-ba198780-cee5-11eb-9504-ada373980421.png)

## Proposal
Hovering over a progress bar where it has an "x" at the end of to cancel an ongoing action should show a tiny box that says "cancel" or something similar to indicate that clicking this "x" would cancel the activity. I don't know, I hope that this explains this correctly. 
Thanks
